rero_ils/modules/acq_orders/jsonschemas/acq_orders/acq_order-v0.0.1.json

```diff
@@ -0,0 +1,235 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Acquisition order",
+  "description": "JSON schema for acq_order.",
+  "additionalProperties": false,
+  "propertiesOrder": [
+    "library",
+    "vendor",
+    "library_order_number",
+    "order_type",
+    "order_status",
+    "currency",
+    "order_date",
+    "description"
+  ],
+  "required": [
+    "$schema",
+    "pid",
+    "vendor",
+    "library",
+    "library_order_number",
+    "order_type",
+    "order_status",
+    "currency"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate acq_order records against.",
+      "type": "string",
+      "minLength": 9,
```

Comment:
this should be replace by a regex, similar to $ref

Reply:
done